frontend/src/charts/choroplethMap/types.ts

```diff
@@ -111,9 +107,16 @@ export interface MapConfig {
   higherIsBetter?: boolean
 }
```

Comment:
this MapConfig is the one i mentioned above; I think it'd be helpful to use a different name here (in VSCODE you can right click and "Rename Symbol" and it'll just rename all occurrences of this specific variable; better than find and replace because it won't rename the variable that has the exact same name)

I think we have to figure out how to get this one that has scheme as a function to play nice with the old MapConfig that has scheme as a string.